src/Storage/Services/IRegisterService.cs

```diff
@@ -18,9 +18,9 @@ public interface IRegisterService
         /// <summary>
         /// Party lookup
         /// </summary>
-        /// <param name="orgNo">organisation number</param>
         /// <param name="person">f or d number</param>
```

Comment:
Maybe we could go for full sentences in the descriptions as well. "The organisation number to look up" for instance.